### DISCUSSION TITLE
Welcome to mock-api Discussions!

### DISCUSSION THREAD

**Discussion by StanleyMasinde** [opensource254/mock-api#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by conradKE** [comment]
Hello 👋 there am conrad Android developer 😊ask me any questions

**Reply by StanleyMasinde** [reply]
How can we improve this project @azim254 ?

**Comment by mukolweke** [comment]
Hi team, am a web developer with experience in PHP/Laravel, VueJs, NuxtJs. Glad to join the team and hope to share my ideas and meet interesting **#254** devs.

**Reply by StanleyMasinde** [reply]
It is an honor to have your expertise here. I look forward to building stuff with you.
